lib/mixlib/install/product.rb

```diff
@@ -77,6 +81,10 @@ def default_value_for(prop)
           "/opt/#{package_name}"
         when :omnibus_project
           package_name
+        when :downloads_url
+          "https://downloads.chef.io/#{product_key}"
```

Comment:
Actually thinking about this more I don't think we should include a fully qualified URL here but just a boolean flag indicating if the product is displayed on the download site. Maybe just:

```ruby
when :download_site
  true
```

This would allow downloads.chef.io to eventually use this flag in its own logic.